mappings/net/minecraft/server/network/ServerPlayNetworkHandler.mapping

```diff
@@ -56,7 +56,13 @@ CLASS net/minecraft/class_3244 net/minecraft/server/network/ServerPlayNetworkHan
 		ARG 2 listener
 	METHOD method_14370 executeCommand (Ljava/lang/String;)V
 		ARG 1 input
-	METHOD method_14371 validateVehicleMove (DDDFF)Z
+	METHOD method_14371 validateMovement (DDDFF)Z
```

Comment:
Shouldn't this be `isMovementInvalid`?

Reply:
Yes. Fixed in bde1e3b064e5ea6662d4ec92fcd55b23c991e679